### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by robert-barcelona**
Added various Jasmine tests and additions to restaurant app

### CODE REVIEW

File: staff/robert-anthony/spotify-app/scripts/logic.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+var logic = {
```
Comment (giodelabarrera) [inline staff/robert-anthony/spotify-app/scripts/logic.js]: missing
```js
'use strict';
```

File: staff/robert-anthony/spotify-app/styles/main.scss [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+}
+
+.mainContainer {
```
Comment (giodelabarrera) [inline staff/robert-anthony/spotify-app/styles/main.scss]: It's more elegant to use kebab-case in class name
```css
.main-container {
```

File: staff/robert-anthony/spotify-app/scripts/logic.js (hunk 1 above)
Comment (robert-barcelona) [reply staff/robert-anthony/spotify-app/scripts/logic.js]: OK!

File: staff/robert-anthony/spotify-app/styles/main.scss (hunk 2 above)
Comment (robert-barcelona) [reply staff/robert-anthony/spotify-app/styles/main.scss]: Either/or

File: staff/robert-anthony/spotify-app/styles/main.css [hunk 3]
```diff
@@ -33,11 +34,103 @@ body {
...
+  border: 4px dotted white;
+  list-style-type: none;
+  visibility: hidden; }
+  .trackInfoDetail li a {
+    text-decoration: none;
+    cursor: pointer;
+    margin: 0;
+    padding: 0;
+    color: inherit; }
+    .trackInfoDetail li a:hover {
```
Comment (laia89) [inline staff/robert-anthony/spotify-app/styles/main.css]: Take care of the indentation. Is better for readability to separate styles by a break line.

File: staff/robert-anthony/spotify-app/scripts/main.js [hunk 4]
```diff
@@ -184,80 +182,95 @@ RadioButtons.prototype.constructor = RadioButtons;
...
+  }.bind(this));
+};
+
 
 /**/
-function ResultsList(dataArray, cssClass) {
+function ResultsList(cssClass) {
 
-  List.call(this, dataArray, "ul");
+  Panel.call(this, "", "ul");
```
Comment (laia89) [inline staff/robert-anthony/spotify-app/scripts/main.js]: Are you sure you need a Panel for that? Maybe a simple Component will work too

**Pull request merged**